server/api/handlers/assignment_handler.go

```diff
@@ -18,7 +27,18 @@ func GetAssignment(c *fiber.Ctx) error {
 		return fiber.NewError(fiber.StatusBadRequest, "assignmentId is not valid")
 	}
 
-	return c.JSON(services.GetAssignment(id))
+	assignment, err := services.GetAssignment(id)
+
+	if err != nil {
```

Comment:
If we use this test several times, it might be worth making an error handler that handles errors of gorm?

Reply:
in line 89 and all handlers that you are going to handle errors from the database (for example: server/api/handlers/community_handler.go line 26 )

Reply:
I think its fine for now